src/examples/example-cosmos.ts

```diff
@@ -0,0 +1,108 @@
+// run me with: npx tsx src/examples/example-cosmos.ts
```

Comment:
ignore this one, it was started but never finished as we don't support Cosmos protocol but only Terra in the ramp protos